Admin-app/src/components/PatientProfile/index.js

```diff
@@ -137,6 +134,24 @@ function PatientProfile() {
       .then((data) => {
         setPatientInfo(data);
         setPriorityFlag(data.flaggedPriority === "1");
+        getStatuses(key).then((statuses) => {
+          statuses &&
+            setPatientInfoStatuses(
+              statuses.map((status) =>
+                createData(
+                  status?.timestamp?.toDate()?.toLocaleString() || "",
+                  status.fever || "No",
+                  status.cough || "No",
+                  status.runnyNose || "No",
+                  status.muscleAche || "No",
+                  status.tiredness || "No",
```

Comment:
When the symptoms modal was created on the patient list, we replaced tiredness with sorethroat. So get the soreThroat field and change the table header to sore throat as well.

Reply:
Corrected